### PR TITLE
Change line 171

### PR DESCRIPTION
**Pull request by Zagalskiy**
Hist color change

### CODE REVIEW

File: ml_web_app.py [hunk 1]
```diff
@@ -168,6 +168,6 @@ def update_query_params():  # Обновление параметра запро
         y=alt.Y("pickups:Q"),
         tooltip=["minute", "pickups"],
     )
-    .configure_mark(opacity=0.3, color="blue"),
+    .configure_mark(opacity=0.3, color="purple"),
```
Comment (gigabyte727) [inline ml_web_app.py:171]: Yes, I agree, I like it.

Comment (gigabyte727) [reply ml_web_app.py:171]: :)

**Review by Tachat-A — APPROVED** [review]
OK!

**Review by gigabyte727 — APPROVED** [review]
Approve! :D

**Pull request merged**